### ISSUE TITLE
scroll event handler

### ISSUE THREAD

**Issue by GeorgMeyer23** [interactive-pioneers/iptools-jquery-anchornav#1]
Assigning a handler to the scroll event may affect performance, because the event is fired for every scrolled pixel/unit.

A better solution is to implement a small delay/throttle like this:

```javascript
handleScroll: function(event) {

   var self = event.data;

   var now = Date.now();
   if (now - self.lastScroll > self.settings.throttle) {
      // do your stuff here...
      self.lastScroll = now;
   }
}
```

**Comment by ain** [comment]
:+1:

**Comment by GeorgMeyer23** [comment]
here is an example.
'scroll' is logged when 'scroll' event is fired.
'update' is logged when delay/throttle takes place.
![image](https://cloud.githubusercontent.com/assets/910259/7961428/df57184a-0a07-11e5-910e-3ef2309295f4.png)

**Comment by ain** [comment]
Yup, I've also used throttling to save on performance as well as in AS3 as in JS, good practice if you don't need heavy performance, e.g. gaming or something.
